<launch>
  <arg name="config_file" default="$(find urban_projection)/config/config.yaml" />
  
  <node name="urban_projector" pkg="urban_projection" type="projector" output="screen">
    <param name="config_file" value="$(arg config_file)" />
  </node>
</launch>

